docs/releases/v2020.2.rst

```diff
@@ -0,0 +1,178 @@
+Gluon 2020.2
+=============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+* GL.iNet
+
+  - GL-AR750S
+
+* TP-Link
+
+  - CPE220 (v3)
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+* EnGenius
+
+  - ENS620EXT [#outdoor]_
+
+* Linksys
+
+  - EA6350 (v3)
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+* Cudy
+
+  - WR1000
+
+lantiq-xway
+~~~~~~~~~~~
+
+* NETGEAR
+
+  - DGN3500B
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+* TP-Link
+
+  - TD-W8970
+
+
+x86-legacy [#newtarget]_
+~~~~~~~~~~
+
+* Devices older than the Pentium 4
+
+
+.. [#newtarget]
+  This is a new target in this release
+
+
+.. [#oudoor]
+  This device is supposed to be setup outdoors and will therefore have it's outdoor mode flag automatically enabled.
+
+
+Major changes
+-------------
+
+Device Classes
+==============
+
+Devices not can be categorised into device classes. This device class can determine which features
+as well as packages are installed on the device when building images.
+
+Yurrently there are two classes used in Gluon, tiny and standard. All devices with less than 64M of Memory or
+7M of usable firmware space are assigned to the tiny class.
+
+WPA3-Support for Private WLAN
+=============================
+
+The private WLAN now supports WPA3-SAE key exchange as well as management frame protection (802.11w).
+For this to work, the firmware needs to be built with the wireless-encryption-wpa3 feature.
+
+Enhanced Open on Client Network
+===============================
+
+* 59a4cd63 gluon-respondd: expose OWE clients in nodeinfo
+* 6692095f gluon-client-bridge: add support for OWE encryption
+
+SAE Encrypted Mesh Links
+========================
+* 461d9040 Add gluon-mesh-wireless-sae package
+
+Responsive Status-Page
+======================
+
+The status page design is now responsive and reflows better on mobile devices.
+
+Primary Domain-Code
+===================
+
+The primary domain code is now visible on the node status page as well as in the respondd information
+emitted by the node.
+
+Logging
+=======
+
+- 9b19883d add gluon-logging package
+- b181803a scripts: check_site: add need_number_range check
+
+Peer cleanup in fastd
+=====================
+
+- 3ccf7fdd gluon-mesh-vpn-fastd: clean up peers and groups on update
+
+Bugfixes
+--------
+
+- The WAN MAC address now matches the one defined in OpenWrt if VXLAN is enabled for the selected domain.
```

Comment:
Hmm, I'm sure we have a few more bugfixes... I'll have another look at the history

Reply:
- 3d091912af41 gluon-web-wifi-config: make mesh VIF options depend on outdoor=false
- 22847e49867f gluon-reload: add missing service restarts on domain switching